### PR TITLE
Infrastructure: add custom TXT input file

### PR DESCRIPTION
**Pull request by halterw**

### CODE REVIEW

**Review by tduretz — COMMENTED** [review]
Fine for me!

File: README.md [hunk 1]
```diff
@@ -154,8 +154,14 @@ Makefile related to cmake is located in a root directory
 To build library, tests and executables with OpenMP and in optimised mode:
 
 ```bash
-make build SET=RiftingPauline
+make build SET=RiftingChenin
 ```
+Specify an .txt input file from SETS folder with TXT:
```
Comment (kulakovri) [inline README.md]: To specify an alternative .txt...

Comment (kulakovri) [reply README.md]: ...folder with TXT parameter

File: README.md [hunk 2]
```diff
@@ -154,8 +154,14 @@ Makefile related to cmake is located in a root directory
...
+
+```bash
+make build SET=RiftingChenin TXT=RiftingChenin_specific.txt
```
Comment (kulakovri) [inline README.md]: RiftingChenin_alternative.txt

**Review by kulakovri — CHANGES_REQUESTED** [review]

**Review by kulakovri — COMMENTED** [review]
It's good, but would worth to have a second look on readme.md
@halterw you can merge it nevertheless

**Comment by halterw** [comment]
Hi @kulakovri, thanks, I've applied your suggestions. Feel free to reformulate it. :) 
Also, I've noticed that in the README.md the chapter about cmake is redundant and I feel that parts of it might need some reviewing and editing.

**Review by kulakovri — APPROVED** [review]

**Comment by kulakovri** [comment]
@halterw feel free to merge it

**Pull request merged**